src/Symfony/Component/Translation/Tests/Formatter/IntlMessageFormatterTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Tests\Formatter;
+
+use Symfony\Component\Translation\Formatter\IntlMessageFormatter;
+
+class IntlMessageFormatterTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        if (!extension_loaded('intl')) {
+            $this->markTestSkipped(
+              'The Intl extension is not available.'
+            );
+        }
+    }
+
+    /**
+     * @dataProvider provideDataForFormat
+     */
+    public function testFormat($expected, $message, $arguments)
+    {
+        $this->assertEquals($expected, trim($this->getMessageFormatter()->format($message, 'en', $arguments)));
+    }
+
+    public function testFormatWithNamedArguments()
+    {
+        if (PHP_VERSION_ID < 50500 || version_compare(INTL_ICU_VERSION, '4.8', '<')) {
```

Comment:
Checking for php version is redundant

https://github.com/symfony/symfony/blob/f557f943babf2893442ceb240521ef1bde785c50/composer.json#L19